### ISSUE TITLE
Add new non-semantic sort options to all scope tabs

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#396]
**Problem Description:**
New Sorting options are desired for each Object, Works, People/Group, Places, Concepts and Events tab. The new sorting options are semantic or non semantic (NS) and can be found [here](https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0).
This ticket will focus on the semantic sort options. 

**Expected Behavior/Solution:**
Object, Works, People/Groups, Concepts and Event tab will contain new semantic search options.
All new options will have an ascending/descending option.
All options will heavily rely on SortName with the expectation there isn't 100% consistency in the record.

**Requirements:**
Those which are **highlighted green** identify the non-semantic sort options to be added & the order which they should go in: 
https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22spreading%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22canvas%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=spreading+canvas (sort toggle)

**Dependencies/Blocks:**
- Blocked By: https://github.com/project-lux/lux-marklogic/issues/342

**Related Github Issues:**
- Semantic Sort: https://github.com/project-lux/lux-frontend/issues/80 

**Related links:**
- https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0

**Wireframe/Mockup:**

**Objects**
![Image](https://github.com/user-attachments/assets/8176c3b7-5fd1-45b1-90b9-d479a0ee5c34)
![Image](https://github.com/user-attachments/assets/3e3d1270-b5fb-4168-8433-5ade66a6d5d1)
**Works**
![Image](https://github.com/user-attachments/assets/572698b5-4bf8-4725-abf3-6a7449808da7)
![Image](https://github.com/user-attachments/assets/817a3363-397a-4c01-be41-801e2de3d367)
**People/Org**
![Image](https://github.com/user-attachments/assets/bf7abdc2-b8d2-4f1e-8c0b-79fd343b0ae8)
![Image](https://github.com/user-attachments/assets/1d5fa429-4829-4e15-9042-34d0ab00fbb9)\
**Places**
![Image](https://github.com/user-attachments/assets/c21f3bbe-6b6b-4f98-8b2f-4287b7e68935)
**Concepts**
![Image](https://github.com/user-attachments/assets/a195e0b5-3e32-480c-8895-aae4939de321)
**Events**
![Image](https://github.com/user-attachments/assets/dbe16172-cc1a-4b86-b485-8fedd1776863)
![Image](https://github.com/user-attachments/assets/8527fff2-2a23-41ff-854c-210624151426)

**Comment by roamye** [comment]
@miketullo95 - can you add the mockups from https://github.com/project-lux/lux-frontend/issues/80 onto this issue? I tried to do it myself but it kept giving me a user error.

**Comment by roamye** [comment]
Objects: 
- Order of options is not correct | https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22c%22%2C%22_lang%22%3A%22en%22%7D&sq=c 

Works: 
- Order of options is not correct | https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22text%22%3A%22c%22%2C%22_lang%22%3A%22en%22%7D&sq=c&ws=workHasDigitalImage%3Adesc

People/Orgs + Places + Concepts: 
- Order of options is not correct 

Events: 
- Order of options is not correct. 
- has digital image does not show, but I believe this is because none of the events have an image.

**Comment by roamye** [comment]
@clarkepeterf - based on this issue and https://github.com/project-lux/lux-frontend/issues/80, is there a reason why certain options do not show? I assume it is because it is not in the data, but wanted to confirm as I noticed Objects had some missing that do show in the data.

**Comment by roamye** [comment]
Approved by UAT - no need for the option `has digital image` to be added in the sort. 
Hot fix for the ordering of the options. See the order listed in: https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0

**Comment by roamye** [comment]
Closing options are all available in PROD. 

New issue has been made to tackle sorting order: https://github.com/project-lux/lux-frontend/issues/480
